Dockerfile

```diff
@@ -16,6 +16,19 @@ RUN npm install
 COPY . /usr/src/app
 RUN npm install
 RUN npm run build
+RUN apk --update add git jq
+RUN version="$(npm -j ls tslint | jq -r .dependencies.tslint.version)" && \
+  printf "Pulling docs from TSLint %s\n" "$version" && \
+  git clone --quiet --branch "$version" https://github.com/palantir/tslint.git && \
+  cd tslint && \
+  npm install && \
+  npm run compile && \
+  npm run docs && \
+  cd .. && \
+  mkdir -p ./lib/docs && \
+  cp --recursive tslint/docs/_data/rules.json ./lib/docs/rules.json && \
+  rm -rf tslint
+RUN apk del git jq
```

Comment:
Oh whoa, good to know! I ended up copying over that other dockerfile you sent to me a bit ago, since it was a bit cleaner. LMK what you think!